HiddenField/cardano-ledger-node-js-api
# IOHKLE-215: Production API tests

### Changes

* Added additional base58 tests to cover 124/248 byte boundaries
* Ensure some stress tests run in serial rather than parallel as they require reset/counters.
* Run prod API stress tests only if we're running heedlessly (run with `npm run ada-api-test -- --headless`).
* Check for index max exceeded in signing on JS side as this will overflow the buffer, so we need to catch it early.
* Ensure timeouts are correct on mocha tests. Prod API tests get 10 seconds each to allow for UI interaction.
* Updated README.md to reflect tests
